server/routes/class.ts

```diff
@@ -0,0 +1,98 @@
+import express, { Request, Response } from 'express'
```

Comment:
The name of this .ts file must be "classes", if we follow the same philosophy of the "materials" file which is already revised